### ISSUE TITLE
inosine & guanosine terms

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6203]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/402573/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done

**Comment by pnrobinson** [comment]
not yet added -- Protege issue?

**Comment by pnrobinson** [comment]
done
